SU-SWS/ace-sdssgryphon
# SDSS-512: updating font sizes to rems

# READY FOR REVIEW

# Summary
- Removing `px` and replacing with `rem`

# Review By (Date)
- 3/8/23

# Urgency
- Normal

# Steps to Test

1. Checkout this branch
2. Clear caches.
3. Verify that the size renders out in the correct `px` listed in ticket.

# Affected Projects or Products
- SDSS Subtheme

# Associated Issues and/or People
[- SDSS-512](https://stanfordits.atlassian.net/browse/SDSS-512)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
